summary.Rmd

```diff
@@ -0,0 +1,41 @@
+Summary
+=======
+
+1. Use available tools
+    a. Image registries (e.g. Docker Hub),
```

Comment:
I'd keep the registries point to the last rule - I'm not sure I'd consider a registry a "tool" in the way we are talking about them (to build, not to store images).

Reply:
When the user is reading, they will be in a chronological mindset, e.g.,

1. build (possibly with tools like linters, cookie cutters, etc.)
2. Document, verison control, etc. (middle steps)
3. Archive and future usage (this is where I think registries are relevant).

It's not wrong to say a registry is a tool, but I think it belongs better where we (also) have it, at the end.

Reply:
Ah ok, I meant in the sense of pulling  and building upon ready-made images from public registries as a summary of:
> First, you want to determine if there is an already existing container that you can use, and in this case, use it and add to your workflow documentation instructions for doing so.

Maybe this would be better:
> Use ready-made images (e.g. from Docker Hub)

Reply:
Oh yes, I misunderstood the point. Yes let's try that and see what others think.

Reply:
As you say, there is a chronological progression through the article, so mentioning it at the outset also means that when the reader comes to the end, the circle of reproducible containers is complete! :)

Reply:
haha, fair enough! We have some framing going on, and maybe that's a good thing :)

Reply:
(cue Lion King music...)